### DISCUSSION TITLE
Saving/loading chat history?

### DISCUSSION THREAD

**Discussion by rubentorresbonet** [langroid/langroid#582] [General]
Apologies if this was asked already, but can an agent keep its chat history among multiple runs? i.e. stop & run the python script again.
Sometimes I want my context to be saved for follow up questions I may have another day.
What options are there for this?
Thanks!

**Comment by pchalasani** [comment]
Thanks @rubentorresbonet for trying out Langroid. I just made an FAQ on this: https://langroid.github.io/langroid/FAQ/#can-i-persist-agent-state-across-multiple-runs

**Reply by rubentorresbonet** [reply]
thank you!!
